default/python/universe_generation/universe_generator.py

```diff
@@ -132,6 +132,7 @@ def create_universe(psd_map):
     seed_rng(seed_pool.pop())
     generate_natives(gsd.native_frequency, systems, home_systems)
 
+
```

Comment:
No need for two blank lines here, guess that somehow slipped in unintentionally...